Formula/d/duplicity.rb

```diff
@@ -349,15 +350,20 @@ class Duplicity < Formula
   end
 
   resource "s3transfer" do
-    url "https://files.pythonhosted.org/packages/3f/ff/5fd9375f3fe467263cff9cad9746fd4c4e1399440ea9563091c958ff90b5/s3transfer-0.7.0.tar.gz"
-    sha256 "fd3889a66f5fe17299fe75b82eae6cf722554edca744ca5d5fe308b104883d2e"
+    url "https://files.pythonhosted.org/packages/a0/b5/4c570b08cb85fdcc65037b5229e00412583bb38d974efecb7ec3495f40ba/s3transfer-0.10.0.tar.gz"
+    sha256 "d0c8bbf672d5eebbe4e57945e23b972d963f07d82f661cabf678a5c88831595b"
   end
 
   resource "setuptools-scm" do
     url "https://files.pythonhosted.org/packages/eb/b1/0248705f10f6de5eefe7ff93e399f7192257b23df4d431d2f5680bb2778f/setuptools-scm-8.0.4.tar.gz"
     sha256 "b5f43ff6800669595193fd09891564ee9d1d7dcb196cab4b2506d53a2e1c95c7"
   end
 
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
+
```

Comment:
```suggestion

```

Reply:
This is covered by the `depends_on "six"` line. We vendor `six`.

Reply:
Ah ok. I'd lazily grepped some other formulae and saw also some `six` resource blocks but didn't think to check the list of vendored python deps. Will update this bit to remove the `six` resource.

Reply:
Would it be reasonable for `audit` to flag if someone has vendored `six` and instead recommend the `depends_on "six"`?

Reply:
Potentially. We are going to have a big discussion on Python dependency vendoring at our annual meeting next month, so would be better to reassess after that.